lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -34,12 +36,13 @@ defmodule Mix.Tasks.Xref do
   """
 
   @switches [compile: :boolean, deps_check: :boolean, archives_check: :boolean,
-             warnings: :boolean, unreachable: :boolean, elixir_version_check: :boolean]
+             warnings: :boolean, unreachable: :boolean, elixir_version_check: :boolean,
+             callers: :string]
 
   @doc """
   Runs this task.
   """
-  @spec run(OptionParser.argv) :: :ok | :error
+  @spec run(OptionParser.argv) :: :ok | :error | [{Path.t, [{atom, atom, non_neg_integer, [pos_integer]}]}]
```

Comment:
Yup, I will just change it to `:ok`. I was trying to follow the pattern of deps.update